.github/workflows/release.yml

```diff
@@ -423,6 +427,12 @@ jobs:
       run: |
         python .github/run_esrp_signing.py deb $env:LINUX_KEY_CODE $env:LINUX_OP_CODE
         python .github/run_esrp_signing.py tar $env:LINUX_KEY_CODE $env:LINUX_OP_CODE
+    
+    - name: Re-name tarball signature file
+      shell: bash
+      run: |
+        signaturepath=$(find signed/*.tar.gz)
+        mv "$signaturepath" "${signaturepath%.tar.gz}.asc"
```

Comment:
My only question is should we not also upload/save the signature too?

Reply:
Do you mean upload to the release? If so, we're already doing that! Here's the dummy release from my test run:

https://github.com/ldennington/git-credential-manager/releases/tag/untagged-4129f63ec90f2c23944a

Reply:
Oh! If we're already uploading the signature then that's great!